CMakeLists.txt

```diff
@@ -29,6 +30,9 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
 include(TargetArch)
 detect_target_architecture(_TARGET_ARCH)
 
+CHECK_INCLUDE_FILE("<getopt.h>" HAVE_GETOPT_H)
+cmake_dependent_option(BASE64_BUILD_CLI "Build the cli for encoding and decoding" ON "HAVE_GETOPT_H" OFF)
+add_feature_info(CLI BASE64_BUILD_CLI "enables the CLI executable for encoding and decoding")
```

Comment:
I'm not sure how this is supposed to work. But when I use cmake-gui it says 
```
-- The following features have been disabled:
* CLI, enables the CLI executable for encoding and decoding
```
but I can not enable CLI, it's not in the list, And it doesn't build base64

Reply:
in the above should it be `... ON "HAVE_GETOPT_H" ON`? That seems to be working.